Makefile

```diff
@@ -853,6 +853,11 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
```

Comment:
You should probably edit the `$(TARBALL)` target as well, to exclude the `tools/remark*`